### ISSUE TITLE
The remote server returned an error: (522).

### ISSUE THREAD

**Issue by wetSuitInCognito** [EasyFarm/EasyFarm#531]
![image](https://github.com/EasyFarm/EasyFarm/assets/136279061/04403f14-2b78-45a8-968e-99ee8e40b1df)
Partially functioned until 6/10.

Observations:
https://www.elitemmonetwork.com/ and any url under this domain do not currently function.

The EliteDlls ping this site perhaps?

**Comment by erwinbal** [comment]
I thought I'm the only one having a same issue. Was working earlier this morning and now getting same error

**Comment by Zirk40** [comment]
Yes exactly the same for me, was working for friend and myself earlier, now getting the 522 error.

**Comment by cabby84** [comment]
same issue here aswell

**Comment by valter3399** [comment]
same here

**Comment by wetSuitInCognito** [comment]
Works again. Resolved 2023-6-12. elitemmonetwork is back up also.

**Comment by Mykezero** [comment]
We'll need to fix the code so that it doesn't reach out to the server every time to see if there's a version update. I'll also add in the feature to select which version of the DLL to use, to support private servers, that may be at different versions of the game.
